src/blocks/AccordionBlock/AccordionBlock.tsx

```diff
@@ -100,8 +100,10 @@ const AccordionContent = styled('div')({
   overflowY: 'hidden',
 
   [TABLET_BP_UP]: {
-    paddingTop: '2rem',
-    paddingBottom: '2rem',
+    padding: '3rem',
+    '& p': {
+      margin: 0,
```

Comment:
Just a question, shall we have some kind of margin-bottom here? Just to separate the paragraphs visually. Or is that intentional?

Reply:
True, I think we have some multi paragraph accordions

Reply:
Ah OK, but maybe not on the last `p` then? Also I realised that the spacing between the `AccordionContent` and the next `AccordionTitle` is smaller in Figma so maybe the titles/headers should only have `paddingBottom` instead of both top and bottom padding?

Reply:
Ah good catch, no `margin-bottom` on the last one 🙌

Reply:
I guess the titles need the same padding top and bottom to align properly when not expanded. So should we maybe decrease the `padding-bottom` on accordion content